frontend/src/i18n/index.ts

```diff
@@ -102,13 +102,56 @@ export const numberFormats: I18nOptions['numberFormats'] = {
   [Locale.ZH_TW]: defaultNumberFormat
 };
 
-export const mapToLocale = (local: string): Locale =>
-  (Object.values(Locale) as string[]).includes(local)
-    ? (local as Locale)
-    : Locale.EN_US;
+export const mapToLocale = (local: string): Locale => {
```

Comment:
can you also fix the typo?


```suggestion
export const mapToLocale = (locale: string): Locale => {
```